src/Bundle/Maker/MakeFactory/DefaultPropertiesGuesserInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Zenstruck\Foundry\Bundle\Maker\MakeFactory;
+
+/**
+ * @internal
+ */
+interface DefaultPropertiesGuesserInterface
```

Comment:
```suggestion
interface DefaultPropertiesGuesser
```

I like to avoid suffixes

Reply:
cool, I don't like `Interface` neither, but people are hard to convince!